### PR TITLE
Add runtime flag on update command

### PR DESCRIPTION
**Pull request by rabingaire**

### CODE REVIEW

File: cmd/utils.go [hunk 1]
```diff
@@ -112,16 +111,17 @@ func getUserInfo(rm *runtime.Manager, client *api.DetaClient) (*runtime.UserInfo
...
+func parseRuntime(runtimeName string) (*runtime.Runtime, error) {
+	newRuntimeName := runtimeName
+	if strings.Contains(runtimeName, runtime.Node) {
+		newRuntimeName = fmt.Sprintf("%s.x", runtimeName)
+	}
 
-		return dateTime.UnixNano() / int64(time.Millisecond), nil
+	progRuntime, err := runtime.CheckRuntime(newRuntimeName)
+	if err != nil {
+		return nil, fmt.Errorf("%s '%s'", err.Error(), runtimeName)
```
Comment (aavshr) [inline cmd/utils.go:123]: if unsupported runtime it will say `run 'deta new --help'` when the command was `deta update`

File: cmd/utils.go [hunk 2]
```diff
@@ -112,16 +111,17 @@ func getUserInfo(rm *runtime.Manager, client *api.DetaClient) (*runtime.UserInfo
 	return u, nil
 }
 
-func parseDateTime(str string) (int64, error) {
```
Comment (aavshr) [inline cmd/utils.go:115]: function was unused?

**Review by aavshr — COMMENTED** [review]
thanks @rabingaire , just two small things/questions.

File: cmd/utils.go (hunk 2 above)
Comment (rabingaire) [reply cmd/utils.go:115]: Yes 👍🏽

**Review by aavshr — APPROVED** [review]

**Pull request merged**